docs/eos-learn/eosio-protocol.md

```diff
@@ -4,7 +4,23 @@ title: EOSIO Protocol
 sidebar_label: EOSIO Protocol
 ---
 
-EOSIO is open-source software that enables launching highly configurable blockchain networks on which developers and entrepreneurs can run highly performant blockchain applications. [EOSIO](https://eos.io/) was created in 2018 and is maintained by [Block One](https://block.one/). The first public network launched on EOSIO is called EOS, also referred as "mainnet". EOS is however just one of multiple public blockchain networks currently using EOSIO. Take a look at the following [EOSIO network list](https://guide.eoscostarica.io/docs/community-resources/eosio-networks). There are also multiple private networks currently using EOSIO.
+## What is EOSIO? 
+
+EOSIO is open-source software that enables the launching of highly configurable blockchain networks on which developers and entrepreneurs can run highly performant blockchain applications. [EOSIO](https://eos.io/) was created in 2018 by [Block One](https://block.one/) (B1). 
+
+EOSIO is software that allows developers to create and run decentralized applications by simulating a computer with resources such as network bandwidth, CPU bandwidth, and RAM. In the same way, EOSIO gives a wide set of tools and protocols for security and permissions for the blockchain networks.
+
+## Differences between EOSIO and EOS
+
+It's important to know what is the difference between EOSIO and EOS to as not to confuse the two concepts, how it was mentioned that EOSIO is the software to develop customized blockchains, in contrast, EOS, also referred to as `mainnet`, corresponds to the first public network launched on EOSIO in the first version.
+
+EOS, in addition to being one of the blockchains deployed on EOSIO also provides the tokens of cryptocurrency used for buying the needed computer resources to run dApps in the EOSIO platform.
```

Comment:
`EOS` is the token symbol used as the mainnet's "system token" that is used to control import aspects of the network such as block producer voting and resource management.